src/components/dashboard/index.jsx

```diff
@@ -4,40 +4,67 @@ import LineChart from '../common/lineChart'
 import Card from './card'
 import { Navbar, Nav, NavDropdown, Container, Row, Col } from 'react-bootstrap'
 // import { LineChart } from './lineChart'
+import './card.css'
 
 export default class Dashboard extends Component {
   state = {}
 
   render() {
+    const indicator = document.querySelector('.nav-indicator')
+    const items = document.querySelectorAll('.nav-item')
+
+    const handleIndicator = (el) => {
+      items.forEach((item) => {
+        item.classList.remove('is-active')
+        item.removeAttribute('style')
+      })
+
+      indicator.style.width = `${el.offsetWidth}px`
+      indicator.style.left = `${el.offsetLeft}px`
+      indicator.style.backgroundColor = el.getAttribute('active-color')
+
+      el.classList.add('is-active')
+      el.style.color = el.getAttribute('active-color')
+    }
+    items.forEach((item, index) => {
+      item.addEventListener('click', (e) => {
+        handleIndicator(e.target)
+      })
+      item.classList.contains('is-active') && handleIndicator(item)
+    })
 
     return (
       <div>
-        <Nav fill variant="tabs" defaultActiveKey="/home">
-          <Nav.Item>
-            <Nav.Link href="/home">TypeForSlack</Nav.Link>
-          </Nav.Item>
-          <Nav.Item>
-            <Nav.Link eventKey="link-1">Practice</Nav.Link>
-          </Nav.Item>
-          <Nav.Item>
-            <Nav.Link eventKey="link-2">Arcade</Nav.Link>
-          </Nav.Item>
-          <Nav.Item>
-            <Nav.Link eventKey="link-3">Challange</Nav.Link>
-          </Nav.Item>
-        </Nav>
+        <nav class="nav">
```

Comment:
When we use HTML elements in React, we use `className` in place of `class`. Refer [this link](https://reactjs.org/docs/dom-elements.html#:~:text=className,use%20the%20class%20attribute%20instead.).